control-plane/src/modules/contract.ts

```diff
@@ -236,9 +236,9 @@ export const contract = c.router({
       clusterId: z.string(),
     }),
   },
-  getFunctionMetrics: {
+  getMetrics: {
     method: "GET",
-    path: "/clusters/:clusterId/services/:serviceName/functions/:functionName/metrics",
+    path: "/clusters/:clusterId/metrics",
```

Comment:
We could instead expose this at each "resource level" if preferred, i.e:
`/clusters/:clusterId/metrics`
`/clusters/:clusterId/metrics/services/:serviceName/metrics`
`/clusters/:clusterId/metrics/services/:serviceName/metrics/functions/:functionName/metrics`